Sources/TuistKit/Models/Target.swift

```diff
@@ -14,7 +14,7 @@ class Target: Equatable {
     let platform: Platform
     let product: Product
     let bundleId: String
-    let infoPlist: AbsolutePath
+    let infoPlist: AbsolutePath?
```

Comment:
Maybe it's worth a small in-line comment to explain why this is an optional

Reply:
Agree